contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,77 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+import logging
+from google import search
+
+def get_google_result(original_content):
+    search_keywords = original_content.split(' ', 1)[1]
+    if search_keywords == '--help':
+        help_message = '''
+                        To use this bot start message with @google followed
+                        by what you want to search for. If found, Zulip will
+                        return the first search result on google.
```

Comment:
Come to think of it, could you add example usage? Two different messages would be enough.

Also, while manually testing the bot I sometimes get a block of text from the link the bot provides and sometimes get only the link - what could be the cause?